BitwardenShared/UI/Auth/CompleteRegistration/CompleteRegistrationProcessor.swift

```diff
@@ -339,3 +354,13 @@ class CompleteRegistrationProcessor: StateProcessor<
         }
     }
 }
+
+// MARK: - CompleteRegistrationDelegate
+
+extension CompleteRegistrationProcessor: MasterPasswordUpdateDelegate {
+    func didUpdateMasterPassword(password: String) {
+        state.passwordText = password
+        state.retypePasswordText = password
```

Comment:
🤔 Maybe something for Product but even though it introduces more friction, shouldn't we let the user retype the password there to ensure they've remembered the password correctly before creating the account?

Reply:
Yeah good call out.  I was following the android implementation of this feature but that doesn't mean its correct either :). For the sake of platform consistency, I think we should keep this as-is and then if we decide to change the behavior then it can be done on both platforms?

Reply:
actually, reading the story bullet it states `fields`.

```
Scenario: Auto-fill generated master password on the choose master password screen
Given I am a user who has allowed the app to generate a master password for me,
When I save the generated password,

Then the generated master password should be automatically filled in the password **fields** on the choose master password screen,

And the password should be masked,

So that I can continue with the account creation process without manually entering the password.
```